tests/publiccloud/aws_cli.pm

```diff
@@ -0,0 +1,63 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Create VM in EC2 using aws binary
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use registration;
+use warnings;
+use testapi;
+use mmapi;
+use strict;
+use utils;
+use publiccloud::utils "select_host_console";
+
+sub run {
+    my ($self, $args) = @_;
+    $self->select_serial_terminal;
+    my $job_id = get_current_job_id();
+
+    add_suseconnect_product 'sle-module-public-cloud';
+    zypper_call 'in aws-cli jq python3-susepubliccloudinfo';
+
+    set_var 'PUBLIC_CLOUD_PROVIDER', 'EC2';
+    my $provider = $self->provider_factory();
+
+    my $image_id = script_output("aws ec2 describe-images --filters 'Name=name,Values=suse-sles-15-sp3-v*-x86_64' 'Name=state,Values=available'  --output=json | jq -r '.Images[] | select( (.Name | contains(\"-ecs\") | not)).ImageId' | head -n1", 240);
```

Comment:
what the purpose of using jq here ? my concern that with such implementation next time aws will change response structure this test may fail

Reply:
@asmorodskyi as we renew our AMIs I can't hardcode it so I need to search for it 'somehow'.
I tried to search for the AMI using `pint` but the AMI I got from `pint` somehow didn't work for me.

Reply:
your testing totally unrelated to images which SUSE shipping to AWS . You testing aws cli . you can query even for Debian or Windows images if it will make test more stable . Main point to see that aws cli is working . And even if you like to stick to SUSE images I don't see any value in verifying what AMIs we actually getting . Even if client will return some random strings instead of valid AMIs this will problem which should not fail *this* test

Reply:
the problem is not in `jq` . Problem in concept called "test isolation" . each test should concentrate on single SUT and not try to cover several things at the same time. Please correct me if I am wrong but I assume SUT of this test is "aws cli" . "aws cli" is interface to Amazon API . "aws cli" does not responsible for validity of the data which Amazon API returning. Here with this dig into response from Amazon you breaking test isolation of test which suppose to concentrate on testing "aws cli" . 

P.S. Keep in mind that I just giving advice how do it better from my POV . You as PR author may accept it or ignore it :) . I am not trying to push you just trying to explain better why I think it should be done differently 
P.P.S. In case you will decide to keep "dig into Amazon API" I actually prefer jq version :)